### DISCUSSION TITLE
Error 404 when generating static pages

### DISCUSSION THREAD

**Discussion by firdausbaihaqi** [nuxt/nuxt#9874] [Questions (Nuxt 2)] [answered]
Hello!!

I was trying to build and generate static pages using npm run generate so that I can deploy it on Netlify, unfortunately, I got this output :

![image](https://user-images.githubusercontent.com/43091431/135740435-04c7db94-22e4-4476-88ed-a7f23ebf3f35.png)

I'm guessing it has something to do with my API endpoint, it's working fine when I build it with SSR mode set to false, but I want to see how it's run on generated static pages, did I miss something?

API endpoint: https://api.nuxtjs.dev/mountains/
link to the project: https://github.com/firdausbaihaqi/nuxt-js-learning

**Comment by renestalder** [comment, marked as answer]
Just a guess, but as you use the built-in `crawler` to let Nuxt find which pages to generate and which not, try to check all your links you have in the application, whether they exist or not. My feeling is the crawler might pickup a link on one of your pages that doesn't exist and tries to generate a page from it. I found that this for example also happened in my case when I have things like `<a href="mailto:...>` or external links (e.g. starting with http(s)).
